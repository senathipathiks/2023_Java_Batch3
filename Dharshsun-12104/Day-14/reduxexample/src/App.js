// import logo from './logo.svg';
import './App.css';
// import Component from '../counter/component';
import { store } from './configure-store';
import {Container} from './counter/container';
import { Provider } from 'react-redux';

function App() {
  return (
    <div className="App">
      {/* <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header> */}
      {/* <Component/> */}
      
      <Provider store={store}>
      <Container/>
      </Provider>

    </div>
  );
}

export default App;
