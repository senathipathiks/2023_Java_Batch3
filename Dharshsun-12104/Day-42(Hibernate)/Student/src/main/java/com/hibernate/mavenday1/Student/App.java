package com.hibernate.mavenday1.Student;

import org.hibernate.SessionFactory;
import org.hibernate.*;
import org.hibernate.cfg.Configuration;

/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
//        System.out.println( "Hello World!" );
//    	Student s1=new Student(3,"Kiruba","Goa");
    	Employee s1 = new Employee(6,"Chandru","Chennai");

    	Configuration cfg=new Configuration().configure("hibernate.cfg.xml");
    	SessionFactory factory=cfg.buildSessionFactory();
    	Session session=factory.openSession();
    	Transaction t=session.beginTransaction();
    	session.save(s1);
    	t.commit();
    	
    }
}
