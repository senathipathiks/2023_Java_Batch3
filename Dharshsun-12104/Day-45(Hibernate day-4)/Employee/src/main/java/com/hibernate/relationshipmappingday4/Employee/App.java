package com.hibernate.relationshipmappingday4.Employee;

import org.hibernate.Session;
import org.hibernate.SessionFactory;
import org.hibernate.Transaction;
import org.hibernate.cfg.Configuration;

/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
//        System.out.println( "Hello World!" );
    	Configuration cfg = new Configuration().configure("hibernate.cfg.xml");
    	SessionFactory factory = cfg.buildSessionFactory();
    	Session session = factory.openSession();
    	Transaction t=session.beginTransaction();
    	Laptop l1 = new Laptop(141,"Lenovo");
    	Employee e1 = new Employee(102,"Dharshsun",l1);
    	session.save(l1);
    	session.save(e1);
    	
    	t.commit();
    	
    }
}
