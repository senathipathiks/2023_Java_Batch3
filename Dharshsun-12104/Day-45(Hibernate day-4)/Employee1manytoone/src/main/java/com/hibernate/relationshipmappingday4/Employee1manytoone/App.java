package com.hibernate.relationshipmappingday4.Employee1manytoone;

import org.hibernate.Session;
import org.hibernate.SessionFactory;
import org.hibernate.Transaction;
import org.hibernate.cfg.Configuration;

/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
//        System.out.println( "Hello World!" );
    	Configuration cfg = new Configuration().configure("hibernate.cfg.xml");
    	SessionFactory factory = cfg.buildSessionFactory();
    	Session session = factory.openSession();
    	Transaction t=session.beginTransaction();
    	Department d1 = new Department(141,"Marketing");
    	Employee e1 = new Employee(102,"Dharshsun",d1);
    	session.save(d1);
    	session.save(e1);
    	
    	t.commit();
    	
    }
}
