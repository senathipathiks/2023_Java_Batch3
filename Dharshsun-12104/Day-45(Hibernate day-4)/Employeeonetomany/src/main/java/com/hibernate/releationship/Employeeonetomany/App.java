package com.hibernate.releationship.Employeeonetomany;

import com.hibernate.dao.CourseDao;
import com.hibernate.dao.InstructorDao;
import com.hibernate.entity.Course;
import com.hibernate.entity.Instructor;

/**
 * Hello world!
 *
 */
public class App 
{
	 public static void main(String[] args) {

	        InstructorDao instructorDao = new InstructorDao();
	        CourseDao courseDao = new CourseDao();

	        Instructor instructor = new Instructor("Ramesh", "Fadatare", "ramesh@javaguides.com");
	        instructorDao.saveInstructor(instructor);

	        // create some courses
	        Course tempCourse1 = new Course("Air Guitar - The Ultimate Guide");
	        tempCourse1.setInstructor(instructor);
	        courseDao.saveCourse(tempCourse1);

	        Course tempCourse2 = new Course("The Pinball Masterclass");
	        tempCourse2.setInstructor(instructor);
	        courseDao.saveCourse(tempCourse2);
	    }
}
