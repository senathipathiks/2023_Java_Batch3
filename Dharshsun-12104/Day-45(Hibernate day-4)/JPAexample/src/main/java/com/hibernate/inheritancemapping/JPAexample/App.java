package com.hibernate.inheritancemapping.JPAexample;

import javax.persistence.*;
public class App 
{
    public static void main( String[] args )
    {
        EntityManager emf = Persistence.createEntityManagerFactory("CourseDB").createEntityManager();
        emf.getTransaction().begin();
        Course c1=new Course(103,"C++","CSE");
        emf.persist(c1);
        emf.getTransaction().commit();
        emf.close();
    }
}
