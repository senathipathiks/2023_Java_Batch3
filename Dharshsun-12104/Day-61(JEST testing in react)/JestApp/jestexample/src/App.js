import logo from './logo.svg';
import './App.css';

function App() {
  return (
    <div className="App">
    <h1>Welcome to my page</h1>
    <br/>
    <h1>Hi welcome to Relevantz</h1>
    </div>
  );
}

export default App;
