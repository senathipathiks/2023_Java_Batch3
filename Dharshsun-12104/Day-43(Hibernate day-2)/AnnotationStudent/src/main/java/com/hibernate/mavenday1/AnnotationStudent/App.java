package com.hibernate.mavenday1.AnnotationStudent;
import java.util.List;
import java.util.Scanner;

import org.hibernate.*;
import org.hibernate.Session;
import org.hibernate.SessionFactory;
import org.hibernate.Transaction;
import org.hibernate.cfg.Configuration;
import com.hibernate.mavenday1.bean.Employee;

/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
    	Scanner sc = new Scanner(System.in);
    	Configuration cfg=new Configuration().configure("Hibernate.cfg.xml");
    	SessionFactory factory=cfg.buildSessionFactory();
    	Session session=factory.openSession();
    	Transaction t=session.beginTransaction();
    	do {
    		System.out.println("1.insert 2.update 3.delete 4.findall 5.find");
    		System.out.println("enter operation number:");
    		int operation = sc.nextInt();
    	
    	switch(operation) {
    	case 1:
    		Employee s1 = new Employee(107, "Vignesh");
        	session.persist(s1);
        	t.commit();	
//        	System.out.println("done");
        	break;
    	case 2:
    		Employee s2 = new Employee();
        	Employee temp = session.find(Employee.class,106);
        	System.out.println(temp);
        	temp.setName("Surya");
        	session.update(temp);
        	t.commit();
        	break;
    	case 3:
    		Employee s3 = new Employee();
    		Employee temp1 = session.find(Employee.class,105);
        	session.delete(temp1);
        	t.commit();
        	break;
    	case 4:
    		List<Employee> empList=session.createQuery("from Employee").list(); //here Employee is class name
        	for(Employee e:empList)
        	{
        		System.out.println(e);
        	}
        	break;
    	case 5:
    		Employee temp2 = session.find(Employee.class,106);
        	System.out.println(temp2);
    	break;
    		
    	}
    	}while(true);
//    	Employee s1 = new Employee(106, "Prem");
//    	Configuration cfg=new Configuration().configure("Hibernate.cfg.xml");
//    	SessionFactory factory=cfg.buildSessionFactory();
//    	Session session=factory.openSession();
//    	Transaction t=session.beginTransaction();
//    	session.persist(s1);
//    	t.commit();
//    	Employee s1 = new Employee();
//    	Employee temp = session.find(Employee.class,105);
//    	System.out.println(temp);
//    	temp.setName("Bala");
//    	session.update(temp);
//    	t.commit();
    	
//    	Employee temp = session.find(Employee.class,105);
//    	session.delete(temp);
//    	t.commit();
//    	List<Employee> empList=session.createQuery("from Employee").list(); //here Employee is class name
//    	for(Employee e:empList)
//    	{
//    		System.out.println(e);
//    	}
    	
    }
  
}
