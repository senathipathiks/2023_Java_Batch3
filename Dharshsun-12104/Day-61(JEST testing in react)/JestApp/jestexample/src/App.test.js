import { render, screen } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer';

describe('Test Suite for app',()=>{
// test.skip('renders learn react link', () => {
//   render(<App/>);
//   const linkElement = screen.getByText("Welcome to my page");
//   expect(linkElement).toBeInTheDocument();
// });
// test('renders welcome to Relevantz',()=>{
//   render(<App/>);
//   const linkElement = screen.getByText("Hi welcome to Relevantz");
//   expect(linkElement).toBeInTheDocument();
// });
test("Snapshot test",() => {
  const comp = renderer.create(<App/>)
  const tree = comp.toJSON();
  expect(tree).toMatchSnapshot();
});
})
// test('two_plus_two',()=>{
//   expect(2+2).toBe(4);
// })
