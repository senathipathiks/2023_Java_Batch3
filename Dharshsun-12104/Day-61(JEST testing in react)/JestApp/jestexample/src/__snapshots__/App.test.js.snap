// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Suite for app Snapshot test 1`] = `
<div
  className="App"
>
  <h1>
    Welcome to my page
  </h1>
  <br />
  <h1>
    Hi welcome to Relevantz
  </h1>
</div>
`;
